config/install/field.field.paragraph.navigation_card.field_customise.yml

```diff
@@ -0,0 +1,22 @@
+langcode: en
+status: true
+dependencies:
+  config:
+    - field.storage.paragraph.field_customise
```

Comment:
Should this field name be something less generic than `customise`? Isn't it `field_show_topic_and_category`?